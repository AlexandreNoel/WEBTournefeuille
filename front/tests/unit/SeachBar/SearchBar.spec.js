import { shallowMount } from '@vue/test-utils'
import SearchBar from '@/components/SearchBar'

describe('SearchBar', () => {
  it('should render properly', async () => {
    const wrapper = shallowMount(SearchBar, {
      propsData: {
        keywords: [
          { id: 1, name: 'KW1' },
          { id: 1, name: 'KW2' },
          { id: 1, name: 'KW3' }
        ],
        value: [{ id: 1, name: 'KW1' }]
      }
    })

    expect(wrapper.html()).toMatchSnapshot()
  })
})
