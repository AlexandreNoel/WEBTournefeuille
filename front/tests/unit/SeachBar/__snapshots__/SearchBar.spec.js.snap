// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar should render properly 1`] = `
<vtoolbar-stub flat="true" scrollthreshold="300" color="transparent">
  <vautocomplete-stub allowoverflow="true" browserautocomplete="off" filter="function _default(item, queryText, itemText) {
                var hasValue = function hasValue(val) {
                    return val != null ? val : '';
                };
                var text = hasValue(itemText);
                var query = hasValue(queryText);
                return text.toString().toLowerCase().indexOf(query.toString().toLowerCase()) > -1;
            }" hidenodata="true" menuprops="[object Object]" value="[object Object]" items="[object Object],[object Object],[object Object]" item-text="name" label="Ce que j'aimerais" multiple="" hide-details="" clearable="" solo=""></vautocomplete-stub>
  <vbtn-stub activeclass="v-btn--active" large="true" tag="button" type="button" color="secondary">J'ai soif !</vbtn-stub>
</vtoolbar-stub>
`;
